GOVERNANCE.md

```diff
@@ -15,7 +15,45 @@ The Node.js Website Team is responsible for the day-to-day technical development
 
 The maintainers on the Node.js Website Team are responsible for steering the technical direction of the Node.js Website, and reserve the right to make final decisions on any issues or pull requests, in line with the Contribution Guidelines, Collaborator Guidelines, the Code of Conduct and the overall Governance premises of the Node.js project.
 
-Members of this team are nominated through the guidelines provided in the Collaborator Guidelines within this repository.
+Members of this team are nominated through the guidelines provided in the Collaborator Guidelines within this repository. After a passed nominations, members should submit a PR to add themselves to the list of current members, shown below.
```

Comment:
this might be a good place for a link to the collaborator guidelines. if i was new and wanted to know more, I'd look for this

and in checking, I realized this is actually in the Contributing

```suggestion
Members of this team are nominated through the guidelines provided in the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md#becoming-a-collaborator) within this repository. After a passed nominations, members should submit a PR to add themselves to the list of current members, shown below.
```